src/application_settings/convenience.py

```diff
@@ -106,3 +109,25 @@ def _parameters_filepath_from_cli(  # pylint: disable=too-many-arguments
         elif settings_class:
             settings_class.set_filepath(universal_cmdline_path, load=load)
     return parser
+
+
+def use_standard_logging(  # pylint: disable=consider-alternative-union-syntax
+    enable: bool = False, fmt: Union[Formatter, None] = None
+) -> None:
+    """Propagate Loguru messages to standard logging"""
+
+    class PropagateHandler(Handler):
+        """Handler to propagate log records to standard logging"""
+
+        def emit(self, record: LogRecord) -> None:
+            """Let the standard logger handle the log record"""
+
+            getLogger(record.name).handle(record)
+
+    handler = PropagateHandler()
+    handler.setFormatter(fmt)
+    logger.remove()  # Remove all handlers added so far, including the default one.
+    logger.add(handler, format="{message}")
+
+    if enable:
+        logger.enable("application_settings")
```

Comment:
There are quite some hard coded strings to `"application_settings"`. Also from a user perspective, does it not make sense to have a global with this name, that also enables users to easily use the `loguru` framework directly (seeing as they can determine our logger name.

Reply:
Done